seed/static/seed/js/services/geocode_service.js

```diff
@@ -19,7 +19,7 @@ angular.module('BE.seed.service.geocode', [])
       };
 
       geocode_factory.confidence_summary = function (property_state_ids, taxlot_state_ids) {
-        return $http.post('/api/v2/geocode/confidence_summary/', {
+        return $http.post('/api/v3/geocode/confidence_summary/', {
```

Comment:
Looks like we're missing the org query param here, which is used for auth. We should be able to add just like the `geocode_by_ids` call above

Reply:
Looks like we still need to add the organization_id query param in this request. I'm testing this by going to the inventory list page, selecting a property, then clicking "Geocode selected" and checking the result in the network tab of devtools.